.github/workflows/release.yml

```diff
@@ -0,0 +1,33 @@
+name: Publish
+
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  build:
+    name: Publish binaries
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install zstd
+        run: sudo apt-get install zstd
+
+      - uses: actions/setup-elixir@v1
+        with:
+          otp-version: '22.2'
+          elixir-version: '1.10.4'
+
+      - name: Build
+        run: export MIX_ENV=prod && mix deps.get && mix release
```

Comment:
I think it would be better if `mix deps.get` was in a separate, cached step. My PR #18 adds this to `test.yml`

Reply:
@polvalente  Is it possible to do `mix deps.get` once and use the cache in all workflows that need it?

Reply:
I don't really know. I'm more familiarized with CircleCI, so I'd have look into Github Actions a bit more. In CircleCI, we have the possibility of running steps one after the other, and some of them conditionally

Reply:
@sreecodeslayer wanted to pitch in with his thoughts, but he's not able to comment here for some weird reason. @Atul9 please check with him on discord about his ideas